src/auth-service.js

```diff
@@ -24,10 +25,9 @@ export {
   logout,
 }
 
-let authTenant = null
+let isInitialized = null
```

Comment:
```suggestion
let isInitialized = false
```
?

Reply:
This is being removed in a later pull request.